yaml/csr.yaml

```diff
@@ -2576,127 +2659,32 @@
-          #- field_name: Hit
-          #  description: >
-          #     Hit indication not supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 20
-          #  lsb: 20
-          #- field_name: Select
-          #  description: >
-          #     Only address matching is supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 19
-          #  lsb: 19
-          #- field_name: Timing
-          #  description: >
-          #     Break before the instruction at the specified address
-          #  type: R
-          #  reset_val: 0
-          #  msb: 18
-          #  lsb: 18
-          #- field_name: Sizelo
-          #  description: >
-          #     Match accesses of any size
-          #  type: R
-          #  reset_val: 0
-          #  msb: 17
-          #  lsb: 16
-          #- field_name: Action
-          #  description: >
-          #     Enter debug mode on match
-          #  type: R
-          #  reset_val: 1
-          #  msb: 15
-          #  lsb: 12
-          #- field_name: Chain
-          #  description: >
-          #     Chaining not supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 11
-          #  lsb: 11
-          #- field_name: Match
-          #  description: >
-          #     Match the whole address
-          #  type: R
-          #  reset_val: 0
-          #  msb: 10
-          #  lsb: 7
-          #- field_name: m
-          #  description: >
-          #     Match in M-mode
-          #  type: R
-          #  reset_val: 1
-          #  msb: 6
-          #  lsb: 6
-          #- field_name: zero
-          #  description: >
-          #     Always zero
-          #  type: R
-          #  reset_val: 0
-          #  msb: 5
-          #  lsb: 5
-          #- field_name: s
-          #  description: >
-          #     S-mode not supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 4
-          #  lsb: 4
-          #- field_name: u
-          #  description: >
-          #     U-mode not supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 3
-          #  lsb: 3
-          #- field_name: execute
-          #  description: >
-          #     Enable matching on instruction address.  Only writeable in Debug mode.
-          #  type: R
-          #  reset_val: 0
-          #  msb: 2
-          #  lsb: 2
-          #- field_name: store
-          #  description: >
-          #     Store address/data matching not supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 1
-          #  lsb: 1
-          #- field_name: load
-          #  description: >
-          #     Load address/data matching not supported
-          #  type: R
-          #  reset_val: 0
-          #  msb: 0
-          #  lsb: 0
+### ENDCOND
+
+
+###############################################################################
+# "tdata2" is temporarily excluded from auto-generation of access testing
```

Comment:
Some difficult warl definitions have simply been excluded from test generation. They should still be in the C macro generation.